drivers/flash/Kconfig.npcx_fiu

```diff
@@ -0,0 +1,35 @@
+# NPCX Flash driver configuration options
+
+# Copyright (c) 2023 Nuvoton Technology Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+config FLASH_NPCX_FIU_QSPI
+	bool "Nuvoton NPCX QSPI Bus Flash driver"
+	default y
+	depends on DT_HAS_NUVOTON_NPCX_FIU_QSPI_ENABLED
+	help
+	  This option enables the QSPI Bus Flash driver for NPCX family of
+	  processors.
+	  Say y if you wish to use QSPI Bus Flash driver on NPCX MCU.
```

Comment:
nit: this line is just a repetition, just drop it :-) same below

Reply:
Done